brayns/common/utils/Utils.cpp

```diff
@@ -90,4 +90,43 @@ std::string replaceFirstOccurrence(std::string input,
         return input;
     return input.replace(pos, toReplace.length(), replaceWith);
 }
+
```

Comment:
Shouldn't this file be lower case for consistency with other ones?

Reply:
Yes, it should